static/nowruz.html

```diff
@@ -170,6 +171,7 @@ <h1>دیوار یادگاری نوروز</h1>
             let selectedColor = _colors[index];
             let lastSelected = null;
             let selectedPixel = null;
+            let clickableCount = 0;
```

Comment:
i think `data.length` ([L188](https://github.com/birlug/birlug.github.io/blob/8c06634a4cd58ecc69a3071a5a8bdfba08a182c2/static/nowruz.html#L188)) would be enough to get the number of colored pixels.